Adding call out for new Discovering Graph Widgets in the Datadog Learning Center (#23862)
DataDog/documentation: content/en/dashboards/_index.md

## Further Reading

> {{< learning-center-callout header="Try Creating Graph Widgets in the Datadog Learning Center" btn_title="Enroll Now" btn_url="https://learn.datadoghq.com/courses/dashboard-graph-widgets">}} You will explore timeseries, query value, toplist, table, distribution, and pie chart widgets. Along the way you’ll get a chance to configure all of these widgets manually and develop an understanding of when each should be utilized. {{< /learning-center-callout >}}

Reviewer: ```suggestion
{{< learning-center-callout header="Try Creating Graph Widgets in the Datadog Learning Center" btn_title="Enroll Now" btn_url="https://learn.datadoghq.com/courses/dashboard-graph-widgets">}} Explore timeseries, query value, toplist, table, distribution, and pie chart widgets. Learn how to configure the widgets and develop an understanding of when each widget type should be utilized. {{< /learning-center-callout >}}
```
